src/vlogger/vlogger.h

```diff
@@ -60,27 +60,53 @@
 #define __log_err(log_fmt, log_args...)          do { VLOG_PRINTF(VLOG_ERROR, log_fmt, ##log_args); } while (0)
 #define __log_warn(log_fmt, log_args...)         do { VLOG_PRINTF(VLOG_WARNING, log_fmt, ##log_args); } while (0)
 #define __log_info(log_fmt, log_args...)         do { VLOG_PRINTF(VLOG_INFO, log_fmt, ##log_args); } while (0)
+
+#if defined(VMA_OPTIMIZE_LOG)
+#define __log_details(log_fmt, log_args...)      ((void)0)
```

Comment:
@igor-ivanov  If --log_details is not on fast path - please consider not excluding it